scripts/scripts/vaccinations/automations/batch/colombia.py

```diff
@@ -0,0 +1,73 @@
+from bs4 import BeautifulSoup
+import requests, pytesseract, pytz, sys
+import pandas as pd
+from datetime import datetime
+try:
+    from PIL import Image
+except ImportError:
+    import Image
+
+def getImageURL(hostname, path):
+    url = hostname + path
+    url_request = requests.get(url)
+    soup = BeautifulSoup(url_request.text, features="lxml")
+    return hostname + soup.find("div", {"class": "interSec1Lchikun left"}).find('img')['src']
+
+def getImgText(imgURL):
+    imgText = pytesseract.image_to_string(
+        Image.open(
+            requests.get(imgURL, stream=True).raw)
+    )
+    return imgText
+
+def main():
+    # Obtains both Image URL and text in Image via tesseract.
+    hostname, path = "https://www.minsalud.gov.co", "/salud/publica/Vacunacion/Paginas/Vacunacion-covid-19.aspx"
+    imgURL = getImageURL(hostname, path)
+    imgText = getImgText(imgURL)
+
+    # Gets the applied doses number 
+    b = imgText.find("DOSIS APLICADAS:")
+    appliedDoses = imgText[(b + 17):(b + 27)]
+
+    # Eliminates all the involved strings if there's any
+    new_vaccinations = []
+    for character in appliedDoses:
+        try:
+            new_vaccinations.append(int(character))
+        except: # easiest way to detect if it's an integer or not, am i right?
+            continue
+    new_vaccinations = int([''.join(str(num) for num in new_vaccinations)][0])
+    # print("Vaccinated: %i" % (new_vaccinations))
```

Comment:
You could also use a regular expression to match digits only:
```suggestion
    new_vaccinations = int(re.sub("[^\d]", "", appliedDoses))
```

But you will need to `import re` library

Reply:
On that.